src/api/general-services/pipeline-status.js

```diff
@@ -241,6 +241,11 @@ const getPipelineStatus = async (experimentId, processName) => {
     },
   };
 
+  if (processName === constants.GEM2S_PROCESS_NAME) {
+    const { paramsHash } = pipelinesHandles[constants.GEM2S_PROCESS_NAME];
+    response[constants.GEM2S_PROCESS_NAME].paramsHash = paramsHash;
```

Comment:
what is this `paramsHash`?

Reply:
This is the property which stores the GEM2S hash, which we check for reruns. It was named like this so past experiments have this too. If we change this, we will have to migrate existing experiments or risk re-runs.

This currently has a different name int the UI (`gem2sHash`). I'll rename it so they have the same name.